packages/rollup-plugin-polyfills-loader/test/snapshots/flattened.html

```diff
@@ -1,8 +1,8 @@
 <html><head>
 
-<link rel="preload" href="./entrypoint-a.js" as="script" crossorigin="anonymous" />
-<link rel="preload" href="shared-4322372e.js" as="script" crossorigin="anonymous" />
 <link rel="preload" href="./entrypoint-b.js" as="script" crossorigin="anonymous" />
+<link rel="preload" href="entrypoint-a.js" as="script" crossorigin="anonymous" />
+<link rel="preload" href="./entrypoint-a.js" as="script" crossorigin="anonymous" />
```

Comment:
This seems like a bug. We shouldnt add duplicate links for the same filepath. Seems like something is going wrong with the handling of the paths.

Reply:
I think we need to change this to normalize the paths and make them consistent:
https://github.com/modernweb-dev/web/blob/718e1524a6e13e04f22f2854657bf3bc00f644a5/packages/rollup-plugin-polyfills-loader/src/rollupPluginPolyfillsLoader.ts#L70-L79

Something like:
```js
        function normalize(path: string) {
          if (path.startsWith('../')) {
            return path;
          } else if (path.startsWith('./')) {
            return path;
          } else if (path.startsWith('/')) {
            return '.' + path;
          } else {
            return './' + path;
          }
        }

        for (const entrypoint of entrypoints) {
          const importPath = normalize(path.posix.relative('', entrypoint.importPath));
          preloaded.push(importPath);

          // js files (incl. chunks) will always be in the root directory
          const pathToRoot = path.posix.dirname(importPath);
          for (const chunkPath of entrypoint.chunk.imports) {
            const relativeChunkPath = normalize(path.posix.join(pathToRoot, chunkPath));
            preloaded.push(relativeChunkPath);
          }
        }

        preloaded = [...new Set(preloaded)];
```

Reply:
that worked perfectly 👍